README.md

```diff
@@ -29,6 +27,14 @@ composer require --dev dealerdirect/phpcodesniffer-composer-installer
 
 That's it.
 
+### Compatibility
+
+This plugin is compatible with both version **1.x** and **2.x** of [Composer][composer], and both version **2.x** and **3.x** of [PHP_CodeSniffer][codesniffer].
+
+PHP **5.x**, **7.x**, and **8.x** are all supported.
+
+For PHP 8 version `0.7.x` (or higher) of this plugin must be _explicitly_ set as  version constraints, as Composer treats minors releases below 1.0 as major releases. In other words: using `^0.6` will not allow Composer to install v0.7 and thus _not_ support PHP8.
```

Comment:
This goes for _both_ Composer 2.x as well as PHP 8.0.

Suggestion for alternative content of this section:

> This plugin is compatible with:
> * PHP 5.x, 7.x and 8.x.
>     Support for PHP 8.0 was added in `v0.7.0`.
> * [Composer][composer] 1.x and 2.x.
>     Support for Composer 2.x was added in `v0.7.0`.
> * [PHP_CodeSniffer][codesniffer] 2.x and 3.x.
>     Support for PHP_CodeSniffer 3.x was added in `v0.4.0`.
>
> Note: as Composer treats minor releases below 1.0.0 as major releases, version `0.7.x` (or higher) of this plugin must be _explicitly_ set as version constraint when using Composer 2.x or PHP 8.0.
> In other words: using `^0.6` will not work with Composer 2.x or PHP 8.0.

Reply:
Will amend.